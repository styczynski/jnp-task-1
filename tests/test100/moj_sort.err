Error in test100/idFile, line 9: xzxx0800
Error in test100/idFile, line 13: xzxx1200
Error in test100/idFile, line 14: xzxx1300
Error in test100/idFile, line 29: xzxx2800
Error in test100/idFile, line 33: xzxx3200
Error in test100/idFile, line 38: xzxx3700
Error in test100/idFile, line 40: xzxx3900
Error in test100/idFile, line 41: xzxx4000
Error in test100/idFile, line 42: xzxx4100
Error in test100/idFile, line 44: xzxx4300
Error in test100/idFile, line 45: xzxx4400
Error in test100/idFile, line 61: xzxx6000
Error in test100/idFile, line 72: xzxx7100
Error in test100/idFile, line 74: xzxx7300
Error in test100/idFile, line 94: xzxx9300
Error in cin, line 1: aa750000
Error in cin, line 2: aa340000
Error in cin, line 3: grupa6/zadanie1/aa240000+xzxx4300
Error in cin, line 4: grupa9/zadanie5/xzxx4100+aa180000+xzxx4100
Error in cin, line 5: grupa9/zadanie2/aam-8100+aam-8100
Error in cin, line 7: aam-6500
Error in cin, line 8: grupa9/zadanie3/aa310000+aa310000
Error in cin, line 10: grupa4/zadanie6/aa240000+xzxx4400+xzxx1200
Error in cin, line 11: aa680000
Error in cin, line 13: aa360000
Error in cin, line 14: aa620000
Error in cin, line 17: grupa4/zadanie2/xzxx7300+xzxx7300
Error in cin, line 18: grupa9/zadanie3/aa010000+aa010000+aam-8200
Error in cin, line 19: grupa6/zadanie5/aa250000+xzxx1300
Error in cin, line 20: grupa7/zadanie3/xzxx7300+xzxx7300+aa350000
Error in cin, line 21: aa360000
Error in cin, line 21: aa360000
Error in cin, line 26: aa890000
Error in cin, line 26: aa890000
Error in cin, line 27: aa330000
Error in cin, line 28: aa310000
Error in cin, line 32: gru pa3/zadanie5/aa950000+aa950000
Error in cin, line 34: aa610000
Error in cin, line 35: grupa9/zadanie5/aa060000+aa870000+aa620000
Error in cin, line 36: aa450000
Error in cin, line 38: aa170000
Error in cin, line 39: grupa7/zadanie6/xzxx9300+aam-4900+xzxx9300
Error in cin, line 41: grupa2/zadanie3/aa670000+xzxx3900
Error in cin, line 43: aa180000
Error in cin, line 45: grupa7/zadanie1/xzxx4300+xzxx4300
Error in cin, line 46: aa340000
Error in cin, line 49: grupa8/zadanie2/xzxx0800+aa160000+aa160000
Error in cin, line 51: grupa9/zadanie3/aa860000+aa860000+aa860000
Error in cin, line 52: grupa2/zadanie6/xzxx6000+aam-6500+xzxx6000
Error in cin, line 54: aa100000
Error in cin, line 55: aa210000
Error in cin, line 56: grupa3/zadanie3/xzxx3700+aa960000+aa960000
Error in cin, line 58: grupa6/zadanie4/xzxx4000+aa460000
Error in cin, line 59: aam-6500
Error in cin, line 60: aam-8100
Error in cin, line 61: grupa5/zadanie2/xzxx3900+xzxx3900
Error in cin, line 62: aa700000
Error in cin, line 63: grupa3/zadanie4/xzxx2800+xzxx2800+aam-7700
Error in cin, line 65: aam-0500
Error in cin, line 66: aa950000
Error in cin, line 68: grupa8/zadanie2/aam-7400+xzxx0800+aa660000
Error in cin, line 69: grupa2/zadanie2/xzxx3900+xzxx3900+xzxx3900
Error in cin, line 71: grupa9/zadanie5/xzxx0800+aam-2600
Error in cin, line 73: aa550000
Error in cin, line 75: aa020000
Error in cin, line 76: aa880000
Error in cin, line 78: grupa5/zadanie1/aa420000+xzxx4000+aam-5300
Error in cin, line 79: aa310000
Error in cin, line 80: aam-5300
Error in cin, line 82: grupa9/zadanie6/aa160000+aam-4900
Error in cin, line 83: aam-5000
Error in cin, line 84: aa960000
Error in cin, line 85: grupa3/zadanie6/aa180000+aa180000+aa420000+aam-2600+aa180000
Error in cin, line 86: grupa9/zadanie6/aam-3000+aam-3000
Error in cin, line 87: grupa9/zadanie5/aa640000+aa270000
Error in cin, line 90: aa180000
Error in cin, line 91: aa610000
Error in cin, line 93: aa210000
Error in cin, line 94: aam-7700
Error in cin, line 94: aam-7700
Error in cin, line 96: grupa4/zadanie5/aa010000+xzxx7100+xzxx7100
Error in cin, line 97: aa760000
Error in cin, line 98: grupa5/zadanie3/xzxx9300+aa270000+xzxx9300
Error in cin, line 99: aam-5000
Error in cin, line 100: grupa6/zadanie3/aa210000+aa210000+xzxx4300
