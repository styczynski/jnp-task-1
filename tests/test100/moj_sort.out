290000;1;
950000;1;
m-6500;2;
